File: src/Entity/CandidatureStage.php
<?php

namespace App\Entity;

use Symfony\Component\Validator\Constraints as Assert;
use App\Repository\CandidatureStageRepository;
use Captcha\Bundle\CaptchaBundle\Form\Type\CaptchaType;
use Doctrine\ORM\Mapping as ORM;
use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\Common\Collections\Collection;
use Captcha\Bundle\CaptchaBundle\Validator\Constraints as CaptchaAssert;



/**
 * @ORM\Entity(repositoryClass=CandidatureStageRepository::class)
 */
class CandidatureStage
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     *
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank
     */
    private $Nom;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank
     */
    private $Prenom;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\Email
     * @Assert\NotBlank
     */
    private $Email;

    /**
     * @ORM\Column(type="integer")
     * @Assert\Positive
     * @Assert\NotBlank
     *
     */
    private $Age;

    /**
     * @ORM\ManyToOne(targetEntity=OffreStage::class, inversedBy="relation",cascade={"persist"})
     * @Assert\NotBlank
     */
    private $id_stage;

    /**
     * @ORM\Column(type="string", length=255, nullable=true)
     */
    private $Commentaire;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNom(): ?string
    {
        return $this->Nom;
    }

    public function setNom(string $Nom): self
    {
        $this->Nom = $Nom;

        return $this;
    }

    public function getPrenom(): ?string
    {
        return $this->Prenom;
    }

    public function setPrenom(string $Prenom): self
    {
        $this->Prenom = $Prenom;

        return $this;
    }

    public function getEmail(): ?string
    {
        return $this->Email;
    }

    public function setEmail(string $Email): self
    {
        $this->Email = $Email;

        return $this;
    }

    public function getAge(): ?int
    {
        return $this->Age;
    }

    public function setAge(int $Age): self
    {
        $this->Age = $Age;

        return $this;
    }

    public function getIdStage(): ?OffreStage
    {
        return $this->id_stage;
    }

    public function setIdStage(?OffreStage $id_stage): self
    {
        $this->id_stage = $id_stage;

        return $this;
    }
    public function __toString()
    {
        return (string) $this->id_stage;
    }

    public function getCommentaire(): ?string
    {
        return $this->Commentaire;
    }

    public function setCommentaire(?string $Commentaire): self
    {
        $this->Commentaire = $Commentaire;

        return $this;
    }
    /**
     * @CaptchaAssert\ValidCaptcha(
     *      message = "CAPTCHA validation failed, try again."
     * )
     */
    protected $captchaCode;

    public function getCaptchaCode()
    {
        return $this->captchaCode;
    }

    public function setCaptchaCode($captchaCode)
    {
        $this->captchaCode = $captchaCode;
    }




}

File: src/Stage/entities/CandidatureStage.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Stage.entities;

/**
 *
 * @author imote
 */
public class CandidatureStage {
    private int id;
    private int id_stage_id;
    private String nom;
    private String prenom;
    private String email;
    private int age;
    private String commentaire;

    public CandidatureStage() {
    }

    public CandidatureStage(int id, int id_stage_id, String nom, String prenom, String email, int age, String commentaire) {
        this.id = id;
        this.id_stage_id = id_stage_id;
        this.nom = nom;
        this.prenom = prenom;
        this.email = email;
        this.age = age;
        this.commentaire = commentaire;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getId_stage_id() {
        return id_stage_id;
    }

    public void setId_stage_id(int id_stage_id) {
        this.id_stage_id = id_stage_id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public String getCommentaire() {
        return commentaire;
    }

    public void setCommentaire(String commentaire) {
        this.commentaire = commentaire;
    }
   
   
    
}